### DISCUSSION TITLE
Please add option to filter waiting lists by categories

### DISCUSSION THREAD

**Discussion by FlohGro-dev** [pretix/pretix#2545] [Ideas]
Currently waiting Lists can only filtered by products, would be great if you would add the possibility to filter by category.

We have Tickets in the same category where it doesn't matter if they e.g. pay 10€ more or not regarding ximum amount of tickets.

**Comment by raphaelm** [comment]
Are you referring to the backend view or the sign-up form for customers?

**Reply by FlohGro-dev** [reply]
I refer to the backend view (a parameter for the answer email to a customer would also be great)
